FizzBuzz/FizzBuzz/Rule.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace FizzBuzz
+{
+    public class Rule
+    {
+        public int number;

```

Comment:
Although technically ok to make these fields, it will be cleaner to make the Properties, e.g. `public int Number {get;set;}`